seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/pom.xml

```diff
@@ -98,6 +98,20 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mock-server</groupId>
+            <artifactId>mockserver-netty-no-dependencies</artifactId>
+            <version>5.14.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.9</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why not use jackson?

Reply:
Because I am familiar with gson, do I need to change it to Jackson?

Reply:
yes please

Reply:
@Hisoka-X done PTAL thks